Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bio"
-version = "0.41.0"
+version = "0.42.0"
```

Comment:
I think the releaser bot is the one that's supposed to take care of incrementing this.

Reply:
Indeed, also the changes in the change log should be removed. The bot does this.